jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/PayloadParser.java

```diff
@@ -72,7 +65,7 @@ public ParserStatus parseFile(BufferedInputStream bis, String fileName) throws F
             status = parseStep(bis, getPayloadLength());
             if (status != ParserStatus.COMPLETE)
                return status;
-            setPayload(removeHeader(Arrays.copyOf(readBuffer, getPayloadLength())));
+            setPayload(stripDot3Header(Arrays.copyOf(readBuffer, getPayloadLength())));
```

Comment:
Could you provide some explanation of this requirements change?

Reply:
The original implementation was the removeHeader function. This needed to remove all headers and only include the payload. We now want to maintain the 1609.2 header if it is a signed header so it will be written into Kafka before it goes to the ADM. We need to be able to record the original header UPER hex so we can utilize it. The signed security header does get removed before the ADM since the ADM doesn't truly support IEEE 1609.2 headers.